src/Core/3DTiles/C3DTBoundingVolume.js

```diff
@@ -143,7 +142,7 @@ class C3DTBoundingVolume {
     }
 
     /**
-     * Checks if the camera is inside the [viewer request volumes](@link https://github.com/CesiumGS/3d-tiles/tree/main/specification#viewer-request-volume).
+     * Checks if the camera is inside the [viewer request volumes](https://github.com/CesiumGS/3d-tiles/tree/main/specification#viewer-request-volume).
```

Comment:
External links as this one do not work. I believe this is due to the fact that doc pages are embedded into the template page, and the links are opened into the embed container, and not in a new page.
I just checked and this is also the case on master so this was not introduced in your PR.

Reply:
The syntax that is working is [text](url)